### PR TITLE
[chore] Refactoring tabscontentcomponent code

### PR DESCRIPTION
**Pull request by wanjikum**
# JIRA TICKET NAME:
[CB-52](https://issues.openmrs.org/browse/CB-52)
Refactor tabsComponent & tabsContentComponent

## SUMMARY:

SOLVED: 

Currently, the tabsComponent & tabsContentComponent are not well structured, the tabsContentComponent is a child component to the tabsComponent but all the implementation for it's props are being implemented in the tabsComponent. It's causing a kind of circular component structure.

I suggest that the implementation for the tabsContentComponent props should reside in the component itself

### CODE REVIEW

**Review by paulupendo — APPROVED** [review]
Good work with the refactor and the tests.

File: app/js/components/tabs/tabContentComponent.jsx [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
+    
+    render(){
+        const {tabs, fetchData, search, addToHistory, getHistory } = this.props;
+        return (
             <div className="tab-content">
-                {props.drawComponent(props.tabs, props.fetchData, props.search, props.addToHistory, props.getHistory)}
+                {this.drawComponent(tabs, fetchData, search, addToHistory, getHistory)}
             </div>
         );
+    };
```
[the remark this answers is not in the document]
Comment (dkayiwa) [reply app/js/components/tabs/tabContentComponent.jsx:39]: @wanjikum did you see the above comment?

[the remark this answers is not in the document]
Comment (wanjikum) [reply app/js/components/tabs/tabContentComponent.jsx:39]: Yes, am working on it now. Thanks.

[the remark this answers is not in the document]
Comment (dkayiwa) [reply app/js/components/tabs/tabContentComponent.jsx:39]: @wanjikum did you address the unnecessary semicolon issue?

[the remark this answers is not in the document]
Comment (wanjikum) [reply app/js/components/tabs/tabContentComponent.jsx:39]: Yes, I addressed @dkayiwa

**Pull request merged**